src/tagmanager/tm_workspace.c

```diff
@@ -836,7 +836,7 @@ find_scope_members (const GPtrArray *tags_array, const gchar *name, TMSourceFile
 
 			/* anonymous type defined in a different file than the variable -
 			 * this isn't the type we are looking for */
-			if (tm_tag_is_anon(test_tag) && (file != test_tag->file || test_tag->file == NULL))
+			if (tm_tag_is_anon(tag) && (file != test_tag->file || test_tag->file == NULL))
```

Comment:
This confuses me and Github draws the indentation strangely.

Anyway, I seems wrong to do s/test_tag/tag/, don't you think?

Reply:
Dammit, I should stop coding today, I start making errors. Sure thing, this is wrong.